# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/1843622.html
# Commit: 1513e7f47215
# Full Hash: 1513e7f47215adb5aeb1182e0856355b991280a5
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2023-07-18 09:25:38
# Description:
#   Bug 1843622 - Better fix / workaround for invalid scale in ScaleOffset r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D183698
# ==============================================================================

diff -r e16e553c6878 -r 1513e7f47215 gfx/tests/crashtests/1843622.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/crashtests/1843622.html	Mon Jul 17 23:52:06 2023 +0000
@@ -0,0 +1,11 @@
+<style>
+#a {
+  transform: scale(0) rotate(0grad);
+  will-change: opacity, contents, left, -webkit-transform;
+}
+:not(mpath) {
+  -webkit-text-stroke-width: 1px;
+}
+</style>
+<textarea id="a" autofocus="autofocus">A</textarea>
+